nanovg.cabal

```diff
@@ -36,16 +36,55 @@ flag gl2
   manual:      True
 
 library
-  exposed-modules:     NanoVG
+  if flag(gles3)
```

Comment:
Can we only put the modules that are actually different in the if clause? It is somewhat hard to see what is actually different here.

Reply:
done